packages/@react-spectrum/searchfield/docs/SearchField.mdx

```diff
@@ -0,0 +1,175 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/searchfield';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import Refresh from '@spectrum-icons/workflow/Refresh';
+import {SearchField} from '@react-spectrum/searchfield';
+```
+
+# SearchField
+
+<p>{docs.exports.SearchField.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['SearchField']} />
+
+## Example
+
+```tsx example
+<SearchField
+  label="Search"
+  placeholder="Enter text"
+  onSubmit={text => alert(`Search for ${text}`)} />
```

Comment:
Any chance we could make this a less annoying example somehow? maybe write it out to a sibling span? i know it makes the example more complicated, but i dislike the alerts

Reply:
ok, done.

i think we should have a mock console or something below the examples.